velox/exec/FilterProject.h

```diff
@@ -53,7 +53,9 @@ class FilterProject : public Operator {
 
   void close() override {
     Operator::close();
-    exprs_->clear();
+    if (exprs_ != nullptr) {
```

Comment:
```
VELOX_CHECK(!initialized_);
```

Reply:
This is a scenario that can actually happen. When ensureInitialize() encounters error, the operator is in unintialized state and system will try to fail the task, clearing the resources.

Reply:
I meant
```
if (exprs_ != nullptr) {
   VELOX_CHECK(!initialized_);
}
```